multicluster/controllers/multicluster/stale_controller.go

```diff
@@ -289,7 +362,16 @@ func (c *StaleResCleanupController) Run(stopCh <-chan struct{}) {
 		go wait.Until(c.runWorker, time.Second, stopCh)
 	}
 
-	<-stopCh
+	ticker := time.NewTicker(5 * time.Minute)
+	defer ticker.Stop()
+	for {
```

Comment:
Will this work? If the error didn't happen in previous step, there will be no worker to handle the queue.

Reply:
In my test it works. But I think you're right. I will do a double check and update the code. Thanks .

Reply:
Moved the goroutine out of the loop.